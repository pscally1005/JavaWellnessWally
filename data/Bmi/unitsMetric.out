Enter 1 to use Imperial units
Enter 2 to use Metric units

Please enter a selection: 
Using Metric units...

