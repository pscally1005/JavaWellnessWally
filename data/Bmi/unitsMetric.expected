Enter 1 to use Imperial units (in,lbs)
Enter 2 to use Metric units (cm,kg)

Please enter a selection: 
Using Metric units...
